docs/troubleshooting-tips.md

```diff
@@ -56,3 +56,7 @@ echo "export PATH=\$PATH:/home/$(whoami)/.local/bin" >> ~/.bashrc
 **A:** You will see this error if there's a route rule with an IP address from a conflicting subnet on the Iptables. The conflicting routes could be either from the host network (bridge mode) or the Kubernetes network. A quick workaround to this issue is to shutdown WSL via the command `wsl --shutdown`. 
 
 **:warning: Please note that shutting down WSL will stop all other distros along with the `rancher-desktop` distro.**
+
+#### Q: How do I fix the `Installation Aborted` error while downgrading from an existing MSI installation to an older EXE version (1.6.x or earlier) on Windows?
+
+**A:** You will see this error if the Windows registry key `HKEY_LOCAL_MACHINE/System/CurrentControlSet/Services/EventLog/Application/RancherDesktopPrivilegedService` that is supposed to be deleted during the MSI uninstallation process is not deleted for some reason. Please manually delete the registry key and try installing the EXE version. You can use Windows Registry Editor, which you can invoke by running the command `regedit` in the Windows Run dialog or from a terminal.
```

Comment:
This would be something like
```
reg.exe delete HKLM\System\CurrentControlSet\Services\EventLog\Application\RancherDesktopPrivilegedService /reg:64
```

Reply:
Provided the command instead of the earlier instruction via GUI

Reply:
A few things to note here, it needs to run in elevated PowerShell if they plan to do this from a command line. Also, it might be good to append `/f` to avoid any additional prompt. e.g 
```
reg.exe delete HKLM\System\CurrentControlSet\Services\EventLog\Application\RancherDesktopPrivilegedService /reg:64 /f
```

Reply:
Yes, I have added in the instructions that users need to run the command in a privileged shell. I have updated the command with the `-f` flag.